<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>474</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>pStream</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../media/Media/我的文档/图标/qt-logo.ico</normaloff>../../../../../media/Media/我的文档/图标/qt-logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>设备:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deviceCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>331</width>
          <height>68</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>331</width>
          <height>68</height>
         </size>
        </property>
        <property name="title">
         <string>设置:</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>10</y>
           <width>281</width>
           <height>60</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="tcpBox">
              <property name="text">
               <string>TCP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="udpBox">
              <property name="text">
               <string>UDP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="icmpBox">
              <property name="text">
               <string>ICMP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="arpBox">
              <property name="text">
               <string>ARP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="ipBox">
              <property name="text">
               <string>IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="etherBox">
              <property name="text">
               <string>Ethernet</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="rawBox">
              <property name="text">
               <string>ShowRawData</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="statisButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>统计</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>停止</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>||</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="resTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>451</width>
        <height>271</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>4512</width>
        <height>271</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="statLabel">
      <property name="text">
       <string>status:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
